playbooks/installation/index.adoc

```diff
@@ -311,7 +311,7 @@ This proposed architecture installs OpenShift via RPM. This is the most common w
 In addition to RPM content, OpenShift requires the ability to pull container images from an external image registry. In order to bring this in house we suggest the following options to build and sync a Standalone Registry.
 
 * A Simple Bootstrap Registry (link:#simple-docker-registry-docker-distribution[See below.])
-* Using the OpenShift Standalone Registry (Coming soon)
+* Using the OpenShift Standalone Registry (link:#syncing-images-using-satellite-6[See below])
```

Comment:
Shouldn't it be this instead?
Syncing image content to Satellite 6 (link:#syncing-images-using-satellite-6[See below])

Reply:
Yes, good catch.